src/CoreML/MLDictionaryFeatureProvider.cs

```diff
@@ -10,10 +10,10 @@
 #if XAMCORE_2_0
```

Comment:
I guess those can be removed, outside the tests ? (later, in a separate PR)

Reply:
My only issue with doing it later in a separate PR is that some of the files are adding NEW #if XAMCORE_2_0 checks at the bottom of the file around the final closing bracket (because the opening bracket is inside the XAMCORE_2_0 around the namespace).  So we'd be adding a new XAMCORE_2_0 check just to go remove it again in a new PR, vs just removing the XAMCORE_2_0 check around the namespace now.

Reply:
This specific one (and any around using statements) isn't really a big deal, but see src/MonoNativeFunctionWrapperAttribute.cs is one example of the above where it's around the namespace and adding new checks at the end of the file.